Packs/CommonScripts/Scripts/SetGridField/README.md

```diff
@@ -11,14 +11,16 @@ Update Grid Table from items or key value pairs.
+| --- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| context_path | Context path to list of items with similar properties or key value pairs.                                                                                                                                                                                                                             |
+| grid_id | Grid ID to modify. This argument can be either: 1) Grid name as it appears in the layout. 2) Grid "Machine name", as can be found in the grid incident field editor under Settings->Advanced->Fields (Incidents).                                                                                     |
+| overwrite | True if to overwrite Grid Data, False otherwise.                                                                                                                                                                                                                                                      |
+| columns | Comma-separated list of grid columns to populate (as appear in the original Grid), for example: (col1,col2,..,coln).                                                                                                                                                                                  |
+| keys | Keys to retrieve from items or &quot;\*&quot; for max keys \(limited when item list to columns amount\) \- Key will not be columns correlated. If you want to leave an empty column, please provide a place holder name that should not be in the context data such as "PLACE_HOLDER"                 |
+| sort_by | Columns names by which to sort the rows.                                                                                                                                                                                                                                                              |
+| unpack_nested_elements | Columns names by which to sort the rows.                                                                                                                                                                                                                                                              |
+| keys_from_nested | Keys to retrieve from nested dictionaries, can be used only when unpack_nested_elements set to False. Keys will not be columns correlated. Default is all keys. **Note**: when the number of values exceeds the number of columns, it truncates the last values that are outside the range for table. |
```

Comment:
```suggestion
| keys_from_nested | Keys to retrieve from nested dictionaries. Can be used only when the unpack_nested_elements argument is set to false. Keys will not be column-correlated. Default is all keys. **Note**: When the number of values exceeds the number of columns, it truncates the last values outside the range for the table. |
```
please update the yml accordingly and check what we discussed about the last note